docs/Installing.asciidoc

```diff
@@ -40,11 +40,34 @@ single command line using the `podman` container engine is the following:
 
 [source,sh]
 ----
-podman run -p 1080:80 -p 1443:443 --rm -it registry.opensuse.org/devel/openqa/containers/openqa-single-instance
+podman run --name openqa --device /dev/kvm -p 1080:80 -p 1443:443 --rm -it \
+  registry.opensuse.org/devel/openqa/containers/openqa-single-instance
 ----
 
 Once the startup has finished, the web UI is accessible via http://localhost:1080
-or https://localhost:1443.
+or https://localhost:1443. You can login without authentication and create a API
+key and secret on http://localhost:1080/api_keys.
+
+You can execute further tools like `openqa-cli` and `openqa-clone-job` via the
+same container. To clone a job from another openQA instance like
+https://openqa.opensuse.org[o3] you can run a command like this:
+[source,sh]
+----
+# make API key and secret available to subsequent commands
+podman exec openqa bash -c \
+  'echo -e "[localhost]\nkey = 1234567890ABCDEF\nsecret = 1234567890ABCDEF" > /etc/openqa/client.conf'
```

Comment:
the bootstrap script already puts key+secret in the config for the worker so we shouldn't overwrite. If in doubt then leave out this code section for now

Reply:
I'll briefly look into https://progress.opensuse.org/issues/155473 because yes, this should not be required (but it was in my tests yesterday which is very weird).